pkg/logger/logger.go

```diff
@@ -0,0 +1,64 @@
+package logger
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+func InitLogger(config *LoggerConfig) *logrus.Logger {
+	l := logrus.New()
+	l.Level = logrus.Level(config.Level)
+	l.SetReportCaller(true)
+
+	logfile := &lumberjack.Logger{
+		Filename:   "./logs/integritySum.log",
+		MaxSize:    5, // MB
+		MaxBackups: 10,
+		MaxAge:     30,   // days
+		Compress:   true, // disabled by default
+	}
+
+	l.SetOutput(io.MultiWriter(logfile, os.Stdout))
+	l.Formatter = &formatter{"[integritySum]"}
+	return l
+}
+
+// Formatter implements logrus.Formatter interface.
+type formatter struct {
+	prefix string
+}
+
+// Format building log message.
+func (f *formatter) Format(entry *logrus.Entry) ([]byte, error) {
+	var sb bytes.Buffer
+
+	var newLine = "\n"
```

Comment:
I don't think we need to check which OS is used by the runtime. We already specified our OS in the docker file. I think that if a user wants to change the OS preferences in the project by changing OS in the docker file, he'll be in need of changing some additional features, and this line also, cause we skip a lot of GOOS types, also macOS is supporting Linux `\r\n`.

Reply:
OK. I understand

Reply:
had to leave. 
`var newLine = "\n"
	if runtime.GOOS == "linux" {
		newLine = "\r\n"
	}
`
because it does not display logs

Reply:
If we set `newLine = "\n"` or `newLine = "\r\n"` ?

Reply:
I understood, I just deleted ` newlines`